### DISCUSSION TITLE
Pictures with "dangerous symbols" like dot, ampersand, slash ...

### DISCUSSION THREAD

**Discussion by kp-org** [jcorporation/myMPD#1135] [General]
I'm using myMPD for a while now and it is the best I have found to fit my needs. Great work.

Unfortunately it seems #1130 breaks to read pictures which have the correct spelling. This is a bit annoying.

Until myMPD 12.0.4 all was fine for me so far, except that I have had one filename with a single quote which wasn't recognized correctly, beside other filenames of pictures with one ore more single quotes were displayed correctly. Not really sure why but I assume it's the length of the filename.

Right now a lot of (Album)Artist arts will no longer displayed. Using underscores in filenames isn't best practice and not really an option. There are better ways to deal with dots, slashes etc. in filenames. And what should be the definition of "dangerous symbols"? Dots, ampersand and many more are valid characters in filenames - even in the Windows world. 

Now, as #1130 was especially for replacing a slash: for AC/DC create  a directory AC and inside that a file DC.jpg. Or replace the slash with use an UTF-8 char (how about U+0B75). 

Please revert this change or make it at least configurable.

Beside that, having GitHub as communication channel only limits potential feedback to GitHub users ... just IMHO.

**Comment by jcorporation** [comment]
I opened a bug for this.

> I'm using myMPD for a while now and it is the best I have found to fit my needs. Great work.

Thanks

> Beside that, having GitHub as communication channel only limits potential feedback to GitHub users ... just IMHO.

What communication channel would you prefer? GitHub has the benefit that there is all at one place.

**Reply by kp-org** [reply]
IMO a channel without an extra login would be helpful, e.g. a mailing list. freelists.org could be a choice, The dokuwiki project uses it and they have some integration with GitHub too.

**Reply by john-** [reply]
The overhead of an additional login is too much?  I think at this point that trying to put that genie back in the bottle is a lost battle.  Then again I still lament we aren't all using Usenet.

**Reply by jcorporation** [reply]
We should discuss this in an extra thread, if there was a demand.
